api/client/ps/custom.go

```diff
@@ -65,7 +65,9 @@ func (c *containerContext) Image() string {
 		return "<no image>"
 	}
 	if c.trunc {
-		return stringutils.Truncate(c.c.Image, 12)
+		if len(c.c.Image) == 64 {
```

Comment:
I don't understand this change. `stringutils.Truncate` truncates strings longer than 12 characters properly.